
файл prm.py в терминале кроме этого сообщения
PS C:\Users\Админ> & C:/Users/Админ/AppData/Local/Microsoft/WindowsApps/python3.11.exe c:/Users/Админ/Desktop/Projpy/prime/prm.py

 ничего не происходит. Но уже не зависает и нет никаких проблемных участков. У меня еще были проблемы с установкой websocket client, поэтому я нашел другой 
коннектор (файл prm_new_connector.py) инструкцию взял отсюда https://sky.pro/media/podklyuchenie-k-baze-dannyh-mysql-s-pomoshhyu-python/

Проблему с строковыми индексами string indices must be integers, not 'str' я возможно решил просто поставив порядковые нужные мне значения ключ-значение VALUES ('[4]', '[5]')
Как везде пишут и сделано у вас VALUES ('{crypto_symbol['p']}') чтобы прописать цикл перед квадратными скобками у меня почему-то не работает.

Итого у меня сейчас ничего не ругается, но ничего и не происходит.
